File: Example/Example/Classes/App/Config/AppConfig.m
//
//  AppConfig.m
//  Example
//
//  Created by wuyong on 2018/11/29.
//  Copyright © 2018 wuyong.site. All rights reserved.
//

#import "AppConfig.h"

@implementation AppConfig

+ (void)refreshController
{
    if (@available(iOS 13.0, *)) {
        FWSceneDelegate *sceneDelegete = (FWSceneDelegate *)UIWindow.fwMainScene.delegate;
        [sceneDelegete setupController];
    } else {
        FWAppDelegate *appDelegate = (FWAppDelegate *)UIApplication.sharedApplication.delegate;
        [appDelegate setupController];
    }
}

@end

File: Example/Example/Classes/App/Config/AppConfig.swift
//
//  AppConfig.swift
//  Example
//
//  Created by wuyong on 2022/5/12.
//  Copyright © 2022 site.wuyong. All rights reserved.
//

import UIKit

class AppConfig {
    
    enum Environment: Int {
        case production = 0
        case staging = 1
        case testing = 2
        case development = 3
    }
    
    static let environment: Environment = {
        #if RELEASE
        .production
        #elseif STAGING
        .staging
        #elseif TESTING
        .testing
        #else
        .development
        #endif
    }()
    
}
